
The following have been reloaded with a version change:
  1) python/2.7.14_openmpi-2.0.1 => python/3.7.0


The following have been reloaded with a version change:
  1) zlib/1.2.11 => zlib/1.2.11_parallel_studio-2017.1


The following have been reloaded with a version change:
  1) hdf5/1.8.18 => hdf5/1.8.18_parallel_studio-2017.1
  2) szip/2.1.1 => szip/2.1.1_parallel_studio-2017.1
  3) zlib/1.2.11_parallel_studio-2017.1 => zlib/1.2.11


The following have been reloaded with a version change:
  1) openmpi/2.0.1_gcc-4.8.5 => openmpi/2.0.1_parallel_studio-2017.1


The following have been reloaded with a version change:
  1) hdf5/1.8.18_parallel_studio-2017.1 => hdf5/1.8.18
  2) openmpi/2.0.1_parallel_studio-2017.1 => openmpi/2.1.2_gcc-4.8.5
  3) python/3.7.0 => python/2.7.14_openmpi-2.1.2
  4) szip/2.1.1_parallel_studio-2017.1 => szip/2.1.1

[argon-lc-i18-21:07925] *** Process received signal ***
[argon-lc-i18-21:07925] Signal: Segmentation fault (11)
[argon-lc-i18-21:07925] Signal code: Address not mapped (1)
[argon-lc-i18-21:07925] Failing at address: 0x38
[argon-lc-i18-21:07925] [ 0] /lib64/libpthread.so.0(+0xf6d0)[0x2ad9510f76d0]
[argon-lc-i18-21:07925] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2ad9723c36ab]
[argon-lc-i18-21:07925] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2ad971acb9da]
[argon-lc-i18-21:07925] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2ad961c8523b]
[argon-lc-i18-21:07925] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2ad961c836b4]
[argon-lc-i18-21:07925] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2ad961c2815c]
[argon-lc-i18-21:07925] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2ad961c594a5]
[argon-lc-i18-21:07925] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2ad9619871d0]
[argon-lc-i18-21:07925] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2ad950dd3629]
[argon-lc-i18-21:07925] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2ad950dd3107]
[argon-lc-i18-21:07925] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2ad950dd5e5d]
[argon-lc-i18-21:07925] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2ad950d4c500]
[argon-lc-i18-21:07925] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2ad950d1a663]
[argon-lc-i18-21:07925] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2ad950d2919d]
[argon-lc-i18-21:07925] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2ad950d1a663]
[argon-lc-i18-21:07925] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2ad950dcc0b7]
[argon-lc-i18-21:07925] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2ad950d33fc8]
[argon-lc-i18-21:07925] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2ad950d1a663]
[argon-lc-i18-21:07925] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2ad950dd0156]
[argon-lc-i18-21:07925] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2ad950dd5e5d]
[argon-lc-i18-21:07925] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2ad950dd5f92]
[argon-lc-i18-21:07925] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2ad950dd55f7]
[argon-lc-i18-21:07925] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2ad950dd5e5d]
[argon-lc-i18-21:07925] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2ad950dd2fec]
[argon-lc-i18-21:07925] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2ad950dd5e5d]
[argon-lc-i18-21:07925] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2ad950dd2fec]
[argon-lc-i18-21:07925] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2ad950dd5e5d]
[argon-lc-i18-21:07925] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2ad950dd2fec]
[argon-lc-i18-21:07925] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2ad950dd5e5d]
[argon-lc-i18-21:07925] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2ad950dd2fec]
[argon-lc-i18-21:07925] *** End of error message ***
[argon-lc-i18-21:07928] *** Process received signal ***
[argon-lc-i18-21:07928] Signal: Segmentation fault (11)
[argon-lc-i18-21:07928] Signal code: Address not mapped (1)
[argon-lc-i18-21:07928] Failing at address: 0x38
[argon-lc-i18-21:07928] [ 0] /lib64/libpthread.so.0(+0xf6d0)[0x2b7387c976d0]
[argon-lc-i18-21:07928] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2b73a8f926ab]
[argon-lc-i18-21:07928] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b73a864d9da]
[argon-lc-i18-21:07928] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b739882523b]
[argon-lc-i18-21:07928] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b73988236b4]
[argon-lc-i18-21:07928] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b73987c815c]
[argon-lc-i18-21:07928] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b73987f94a5]
[argon-lc-i18-21:07928] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b73985271d0]
[argon-lc-i18-21:07928] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b7387973629]
[argon-lc-i18-21:07928] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b7387973107]
[argon-lc-i18-21:07928] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7387975e5d]
[argon-lc-i18-21:07928] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b73878ec500]
[argon-lc-i18-21:07928] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b73878ba663]
[argon-lc-i18-21:07928] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b73878c919d]
[argon-lc-i18-21:07928] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b73878ba663]
[argon-lc-i18-21:07928] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b738796c0b7]
[argon-lc-i18-21:07928] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b73878d3fc8]
[argon-lc-i18-21:07928] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b73878ba663]
[argon-lc-i18-21:07928] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b7387970156]
[argon-lc-i18-21:07928] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7387975e5d]
[argon-lc-i18-21:07928] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b7387975f92]
[argon-lc-i18-21:07928] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b73879755f7]
[argon-lc-i18-21:07928] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7387975e5d]
[argon-lc-i18-21:07928] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7387972fec]
[argon-lc-i18-21:07928] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7387975e5d]
[argon-lc-i18-21:07928] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7387972fec]
[argon-lc-i18-21:07928] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7387975e5d]
[argon-lc-i18-21:07928] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7387972fec]
[argon-lc-i18-21:07928] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7387975e5d]
[argon-lc-i18-21:07928] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7387972fec]
[argon-lc-i18-21:07928] *** End of error message ***
[argon-lc-i18-21:07927] *** Process received signal ***
[argon-lc-i18-21:07927] Signal: Segmentation fault (11)
[argon-lc-i18-21:07927] Signal code: Address not mapped (1)
[argon-lc-i18-21:07927] Failing at address: 0x38
[argon-lc-i18-21:07927] [ 0] /lib64/libpthread.so.0(+0xf6d0)[0x2b828103d6d0]
[argon-lc-i18-21:07927] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2b82a23026ab]
[argon-lc-i18-21:07927] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b82a1a0a9da]
[argon-lc-i18-21:07927] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b8291bcb23b]
[argon-lc-i18-21:07927] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b8291bc96b4]
[argon-lc-i18-21:07927] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b8291b6e15c]
[argon-lc-i18-21:07927] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b8291b9f4a5]
[argon-lc-i18-21:07927] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b82918cd1d0]
[argon-lc-i18-21:07927] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b8280d19629]
[argon-lc-i18-21:07927] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b8280d19107]
[argon-lc-i18-21:07927] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b8280d1be5d]
[argon-lc-i18-21:07927] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b8280c92500]
[argon-lc-i18-21:07927] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b8280c60663]
[argon-lc-i18-21:07927] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b8280c6f19d]
[argon-lc-i18-21:07927] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b8280c60663]
[argon-lc-i18-21:07927] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b8280d120b7]
[argon-lc-i18-21:07927] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b8280c79fc8]
[argon-lc-i18-21:07927] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b8280c60663]
[argon-lc-i18-21:07927] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b8280d16156]
[argon-lc-i18-21:07927] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b8280d1be5d]
[argon-lc-i18-21:07927] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b8280d1bf92]
[argon-lc-i18-21:07927] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b8280d1b5f7]
[argon-lc-i18-21:07927] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b8280d1be5d]
[argon-lc-i18-21:07927] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b8280d18fec]
[argon-lc-i18-21:07927] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b8280d1be5d]
[argon-lc-i18-21:07927] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b8280d18fec]
[argon-lc-i18-21:07927] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b8280d1be5d]
[argon-lc-i18-21:07927] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b8280d18fec]
[argon-lc-i18-21:07927] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b8280d1be5d]
[argon-lc-i18-21:07927] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b8280d18fec]
[argon-lc-i18-21:07927] *** End of error message ***
[argon-lc-i18-21:07926] *** Process received signal ***
[argon-lc-i18-21:07926] Signal: Segmentation fault (11)
[argon-lc-i18-21:07926] Signal code: Address not mapped (1)
[argon-lc-i18-21:07926] Failing at address: 0x38
[argon-lc-i18-21:07926] [ 0] /lib64/libpthread.so.0(+0xf6d0)[0x2accee3706d0]
[argon-lc-i18-21:07926] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2acd0f6a36ab]
[argon-lc-i18-21:07926] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2acd0edab9da]
[argon-lc-i18-21:07926] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2accfeefe23b]
[argon-lc-i18-21:07926] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2accfeefc6b4]
[argon-lc-i18-21:07926] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2accfeea115c]
[argon-lc-i18-21:07926] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2accfeed24a5]
[argon-lc-i18-21:07926] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2accfec001d0]
[argon-lc-i18-21:07926] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2accee04c629]
[argon-lc-i18-21:07926] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2accee04c107]
[argon-lc-i18-21:07926] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2accee04ee5d]
[argon-lc-i18-21:07926] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2accedfc5500]
[argon-lc-i18-21:07926] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2accedf93663]
[argon-lc-i18-21:07926] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2accedfa219d]
[argon-lc-i18-21:07926] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2accedf93663]
[argon-lc-i18-21:07926] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2accee0450b7]
[argon-lc-i18-21:07926] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2accedfacfc8]
[argon-lc-i18-21:07926] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2accedf93663]
[argon-lc-i18-21:07926] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2accee049156]
[argon-lc-i18-21:07926] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2accee04ee5d]
[argon-lc-i18-21:07926] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2accee04ef92]
[argon-lc-i18-21:07926] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2accee04e5f7]
[argon-lc-i18-21:07926] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2accee04ee5d]
[argon-lc-i18-21:07926] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2accee04bfec]
[argon-lc-i18-21:07926] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2accee04ee5d]
[argon-lc-i18-21:07926] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2accee04bfec]
[argon-lc-i18-21:07926] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2accee04ee5d]
[argon-lc-i18-21:07926] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2accee04bfec]
[argon-lc-i18-21:07926] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2accee04ee5d]
[argon-lc-i18-21:07926] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2accee04bfec]
[argon-lc-i18-21:07926] *** End of error message ***
[argon-lc-i18-21:07930] *** Process received signal ***
[argon-lc-i18-21:07929] *** Process received signal ***
[argon-lc-i18-21:07932] *** Process received signal ***
[argon-lc-i18-21:07932] Signal: Segmentation fault (11)
[argon-lc-i18-21:07932] Signal code: Address not mapped (1)
[argon-lc-i18-21:07932] Failing at address: 0x38
[argon-lc-i18-21:07930] Signal: Segmentation fault (11)
[argon-lc-i18-21:07930] Signal code: Address not mapped (1)
[argon-lc-i18-21:07930] Failing at address: 0x38
[argon-lc-i18-21:07929] Signal: Segmentation fault (11)
[argon-lc-i18-21:07929] Signal code: Address not mapped (1)
[argon-lc-i18-21:07929] Failing at address: 0x38
[argon-lc-i18-21:07932] [ 0] /lib64/libpthread.so.0(+0xf6d0)[0x2b1af12b16d0]
[argon-lc-i18-21:07932] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2b1b125c66ab]
[argon-lc-i18-21:07932] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b1b11cce9da]
[argon-lc-i18-21:07932] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b1b01e3f23b]
[argon-lc-i18-21:07932] [ 4] [argon-lc-i18-21:07931] *** Process received signal ***
[argon-lc-i18-21:07930] [ 0] /lib64/libpthread.so.0(+0xf6d0)[0x2b352109c6d0]
[argon-lc-i18-21:07930] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2b35423c36ab]
[argon-lc-i18-21:07930] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b3541acb9da]
[argon-lc-i18-21:07930] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b3531c2a23b]
[argon-lc-i18-21:07930] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b1b01e3d6b4]
[argon-lc-i18-21:07932] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b1b01de215c]
[argon-lc-i18-21:07932] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b3531c286b4]
[argon-lc-i18-21:07930] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b3531bcd15c]
[argon-lc-i18-21:07930] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b3531bfe4a5]
[argon-lc-i18-21:07930] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b353192c1d0]
[argon-lc-i18-21:07930] [ 8] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b1b01e134a5]
[argon-lc-i18-21:07932] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b1b01b411d0]
[argon-lc-i18-21:07932] [ 8] [argon-lc-i18-21:07931] Signal: Segmentation fault (11)
[argon-lc-i18-21:07931] Signal code: Address not mapped (1)
[argon-lc-i18-21:07931] Failing at address: 0x38
[argon-lc-i18-21:07929] [ 0] /lib64/libpthread.so.0(+0xf6d0)[0x2b2d8fcec6d0]
[argon-lc-i18-21:07929] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2b2db0f926ab]
[argon-lc-i18-21:07929] [ 2] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b3520d78629]
[argon-lc-i18-21:07930] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b3520d78107]
[argon-lc-i18-21:07930] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b1af0f8d629]
[argon-lc-i18-21:07932] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b1af0f8d107]
[argon-lc-i18-21:07932] [10] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b2db069a9da]
[argon-lc-i18-21:07929] [ 3] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3520d7ae5d]
[argon-lc-i18-21:07930] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b1af0f8fe5d]
[argon-lc-i18-21:07932] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b1af0f06500]
[argon-lc-i18-21:07932] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b1af0ed4663]
[argon-lc-i18-21:07932] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b3520cf1500]
[argon-lc-i18-21:07930] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b3520cbf663]
[argon-lc-i18-21:07930] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b3520cce19d]
[argon-lc-i18-21:07930] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b2da087a23b]
[argon-lc-i18-21:07929] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b2da08786b4]
[argon-lc-i18-21:07929] [ 5] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b1af0ee319d]
[argon-lc-i18-21:07932] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b1af0ed4663]
[argon-lc-i18-21:07932] [15] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b3520cbf663]
[argon-lc-i18-21:07930] [15] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b2da081d15c]
[argon-lc-i18-21:07929] [ 6] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b3520d710b7]
[argon-lc-i18-21:07930] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b1af0f860b7]
[argon-lc-i18-21:07932] [16] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b2da084e4a5]
[argon-lc-i18-21:07929] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b2da057c1d0]
[argon-lc-i18-21:07929] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b1af0eedfc8]
[argon-lc-i18-21:07932] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b1af0ed4663]
[argon-lc-i18-21:07932] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b3520cd8fc8]
[argon-lc-i18-21:07930] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b3520cbf663]
[argon-lc-i18-21:07930] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b2d8f9c8629]
[argon-lc-i18-21:07929] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b2d8f9c8107]
[argon-lc-i18-21:07929] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b1af0f8a156]
[argon-lc-i18-21:07932] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b1af0f8fe5d]
[argon-lc-i18-21:07932] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b3520d75156]
[argon-lc-i18-21:07930] [19] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b1af0f8ff92]
[argon-lc-i18-21:07932] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b2d8f9cae5d]
[argon-lc-i18-21:07929] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3520d7ae5d]
[argon-lc-i18-21:07930] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b3520d7af92]
[argon-lc-i18-21:07930] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b3520d7a5f7]
[argon-lc-i18-21:07930] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3520d7ae5d]
[argon-lc-i18-21:07930] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b2d8f941500]
[argon-lc-i18-21:07929] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b2d8f90f663]
[argon-lc-i18-21:07929] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b2d8f91e19d]
[argon-lc-i18-21:07929] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b1af0f8f5f7]
[argon-lc-i18-21:07932] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b1af0f8fe5d]
[argon-lc-i18-21:07932] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b1af0f8cfec]
[argon-lc-i18-21:07932] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b1af0f8fe5d]
[argon-lc-i18-21:07932] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3520d77fec]
[argon-lc-i18-21:07930] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3520d7ae5d]
[argon-lc-i18-21:07930] [25] [argon-lc-i18-21:07931] [ 0] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b1af0f8cfec]
[argon-lc-i18-21:07932] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b1af0f8fe5d]
[argon-lc-i18-21:07932] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b2d8f90f663]
[argon-lc-i18-21:07929] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b2d8f9c10b7]
[argon-lc-i18-21:07929] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3520d77fec]
[argon-lc-i18-21:07930] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3520d7ae5d]
[argon-lc-i18-21:07930] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b2d8f928fc8]
[argon-lc-i18-21:07929] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b2d8f90f663]
[argon-lc-i18-21:07929] [18] /lib64/libpthread.so.0(+0xf6d0)[0x2b3bab1ca6d0]
[argon-lc-i18-21:07931] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm2.so(ompi_mtl_psm2_send+0x19b)[0x2b3bcc48f6ab]
[argon-lc-i18-21:07931] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b3bcbb979da]
[argon-lc-i18-21:07931] [ 3] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b1af0f8cfec]
[argon-lc-i18-21:07932] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b1af0f8fe5d]
[argon-lc-i18-21:07932] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3520d77fec]
[argon-lc-i18-21:07930] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3520d7ae5d]
[argon-lc-i18-21:07930] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3520d77fec]
[argon-lc-i18-21:07930] *** End of error message ***
/opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b1af0f8cfec]
[argon-lc-i18-21:07932] *** End of error message ***
/opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b2d8f9c5156]
[argon-lc-i18-21:07929] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b2d8f9cae5d]
[argon-lc-i18-21:07929] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b3bbbd5823b]
[argon-lc-i18-21:07931] [ 4] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b2d8f9caf92]
[argon-lc-i18-21:07929] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b2d8f9ca5f7]
[argon-lc-i18-21:07929] [22] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b3bbbd566b4]
[argon-lc-i18-21:07931] [ 5] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b2d8f9cae5d]
[argon-lc-i18-21:07929] [23] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b3bbbcfb15c]
[argon-lc-i18-21:07931] [ 6] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b2d8f9c7fec]
[argon-lc-i18-21:07929] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b2d8f9cae5d]
[argon-lc-i18-21:07929] [25] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b3bbbd2c4a5]
[argon-lc-i18-21:07931] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b3bbba5a1d0]
[argon-lc-i18-21:07931] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b2d8f9c7fec]
[argon-lc-i18-21:07929] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b2d8f9cae5d]
[argon-lc-i18-21:07929] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b2d8f9c7fec]
[argon-lc-i18-21:07929] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b3baaea6629]
[argon-lc-i18-21:07931] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b2d8f9cae5d]
[argon-lc-i18-21:07929] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b2d8f9c7fec]
[argon-lc-i18-21:07929] *** End of error message ***
/opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b3baaea6107]
[argon-lc-i18-21:07931] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3baaea8e5d]
[argon-lc-i18-21:07931] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b3baae1f500]
[argon-lc-i18-21:07931] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b3baaded663]
[argon-lc-i18-21:07931] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b3baadfc19d]
[argon-lc-i18-21:07931] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b3baaded663]
[argon-lc-i18-21:07931] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b3baae9f0b7]
[argon-lc-i18-21:07931] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b3baae06fc8]
[argon-lc-i18-21:07931] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b3baaded663]
[argon-lc-i18-21:07931] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b3baaea3156]
[argon-lc-i18-21:07931] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3baaea8e5d]
[argon-lc-i18-21:07931] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b3baaea8f92]
[argon-lc-i18-21:07931] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b3baaea85f7]
[argon-lc-i18-21:07931] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3baaea8e5d]
[argon-lc-i18-21:07931] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3baaea5fec]
[argon-lc-i18-21:07931] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3baaea8e5d]
[argon-lc-i18-21:07931] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3baaea5fec]
[argon-lc-i18-21:07931] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3baaea8e5d]
[argon-lc-i18-21:07931] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3baaea5fec]
[argon-lc-i18-21:07931] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b3baaea8e5d]
[argon-lc-i18-21:07931] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b3baaea5fec]
[argon-lc-i18-21:07931] *** End of error message ***
[argon-itf-cf46-32:04428] *** Process received signal ***
[argon-itf-cf46-32:04428] Signal: Segmentation fault (11)
[argon-itf-cf46-32:04428] Signal code: Address not mapped (1)
[argon-itf-cf46-32:04428] Failing at address: 0x2b7f0a3b5f88
[argon-itf-cf46-32:04428] [ 0] /lib64/libc.so.6(+0x362f0)[0x2b7ef99a12f0]
[argon-itf-cf46-32:04428] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm.so(ompi_mtl_psm_send+0xb8)[0x2b7f1a5694d8]
[argon-itf-cf46-32:04428] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b7f1980a9da]
[argon-itf-cf46-32:04428] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b7f09be323b]
[argon-itf-cf46-32:04428] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b7f09be16b4]
[argon-itf-cf46-32:04428] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b7f09b8615c]
[argon-itf-cf46-32:04428] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b7f09bb74a5]
[argon-itf-cf46-32:04428] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b7f098e51d0]
[argon-itf-cf46-32:04428] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b7ef8d31629]
[argon-itf-cf46-32:04428] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b7ef8d31107]
[argon-itf-cf46-32:04428] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7ef8d33e5d]
[argon-itf-cf46-32:04428] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b7ef8caa500]
[argon-itf-cf46-32:04428] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b7ef8c78663]
[argon-itf-cf46-32:04428] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b7ef8c8719d]
[argon-itf-cf46-32:04428] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b7ef8c78663]
[argon-itf-cf46-32:04428] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b7ef8d2a0b7]
[argon-itf-cf46-32:04428] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b7ef8c91fc8]
[argon-itf-cf46-32:04428] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b7ef8c78663]
[argon-itf-cf46-32:04428] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b7ef8d2e156]
[argon-itf-cf46-32:04428] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7ef8d33e5d]
[argon-itf-cf46-32:04428] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b7ef8d33f92]
[argon-itf-cf46-32:04428] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b7ef8d335f7]
[argon-itf-cf46-32:04428] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7ef8d33e5d]
[argon-itf-cf46-32:04428] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7ef8d30fec]
[argon-itf-cf46-32:04428] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7ef8d33e5d]
[argon-itf-cf46-32:04428] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7ef8d30fec]
[argon-itf-cf46-32:04428] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7ef8d33e5d]
[argon-itf-cf46-32:04428] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7ef8d30fec]
[argon-itf-cf46-32:04428] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b7ef8d33e5d]
[argon-itf-cf46-32:04428] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.[argon-itf-cf46-32:04429] *** Process received signal ***
[argon-itf-cf46-32:04429] Signal: Segmentation fault (11)
[argon-itf-cf46-32:04429] Signal code: Address not mapped (1)
[argon-itf-cf46-32:04429] Failing at address: 0x1
[argon-itf-cf46-32:04429] [ 0] /lib64/libc.so.6(+0x362f0)[0x2b4c84b802f0]
[argon-itf-cf46-32:04429] [ 1] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_mtl_psm.so(ompi_mtl_psm_send+0xb8)[0x2b4ca575c4d8]
[argon-itf-cf46-32:04429] [ 2] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/openmpi/mca_pml_cm.so(+0x29da)[0x2b4ca49fd9da]
[argon-itf-cf46-32:04429] [ 3] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_sendrecv_nonzero_actual+0x17b)[0x2b4c94dc223b]
[argon-itf-cf46-32:04429] [ 4] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_coll_base_allgather_intra_bruck+0xd4)[0x2b4c94dc06b4]
[argon-itf-cf46-32:04429] [ 5] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(ompi_comm_split+0xbc)[0x2b4c94d6515c]
[argon-itf-cf46-32:04429] [ 6] /opt/apps/openmpi/2.1.2_gcc-4.8.5/lib/libmpi.so.20(MPI_Comm_split+0xa5)[0x2b4c94d964a5]
[argon-itf-cf46-32:04429] [ 7] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/python2.7/site-packages/mpi4py/MPI.so(+0x1121d0)[0x2b4c94ac41d0]
[argon-itf-cf46-32:04429] [ 8] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6f79)[0x2b4c83f10629]
[argon-itf-cf46-32:04429] [ 9] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x6a57)[0x2b4c83f10107]
[argon-itf-cf46-32:04429] [10] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b4c83f12e5d]
[argon-itf-cf46-32:04429] [11] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x86500)[0x2b4c83e89500]
[argon-itf-cf46-32:04429] [12] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b4c83e57663]
[argon-itf-cf46-32:04429] [13] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(+0x6319d)[0x2b4c83e6619d]
[argon-itf-cf46-32:04429] [14] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b4c83e57663]
[argon-itf-cf46-32:04429] [15] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_CallObjectWithKeywords+0x47)[0x2b4c83f090b7]
[argon-itf-cf46-32:04429] [16] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyInstance_New+0x68)[0x2b4c83e70fc8]
[argon-itf-cf46-32:04429] [17] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyObject_Call+0x43)[0x2b4c83e57663]
[argon-itf-cf46-32:04429] [18] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x3aa6)[0x2b4c83f0d156]
[argon-itf-cf46-32:04429] [19] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b4c83f12e5d]
[argon-itf-cf46-32:04429] [20] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCode+0x32)[0x2b4c83f12f92]
[argon-itf-cf46-32:04429] [21] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x8f47)[0x2b4c83f125f7]
[argon-itf-cf46-32:04429] [22] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b4c83f12e5d]
[argon-itf-cf46-32:04429] [23] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b4c83f0ffec]
[argon-itf-cf46-32:04429] [24] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b4c83f12e5d]
[argon-itf-cf46-32:04429] [25] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b4c83f0ffec]
[argon-itf-cf46-32:04429] [26] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b4c83f12e5d]
[argon-itf-cf46-32:04429] [27] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalFrameEx+0x693c)[0x2b4c83f0ffec]
[argon-itf-cf46-32:04429] [28] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEval_EvalCodeEx+0x80d)[0x2b4c83f12e5d]
[argon-itf-cf46-32:04429] [29] /opt/apps/python/2.7.14_openmpi-2.1.2/lib/libpython2.7.so.1.0(PyEvso.1.0(PyEval_EvalFrameEx+0x693c)[0x2b7ef8d30fec]
[argon-itf-cf46-32:04428] *** End of error message ***
al_EvalFrameEx+0x693c)[0x2b4c83f0ffec]
[argon-itf-cf46-32:04429] *** End of error message ***
